dispatcher/MongoConnect.py

```diff
@@ -77,6 +76,10 @@ def __init__(self, config, daq_config, log, control_mc, runs_mc, hypervisor, tes
         # Which control keys do we look for?
         self.control_keys = config['ControlKeys'].split()
 
+        # a place to buffer commands temporarily
+        self.command_queue = []
+        self.q_mutex = threading.Mutex()
```

Comment:
I don't know this command, so might need some help here, also my python doesn't know it.

Reply:
In python it's apparently a `Lock`, but in every other language it's a mutex.